x-pack/packages/kbn-slo-schema/src/rest_specs/slo.ts

```diff
@@ -269,6 +276,7 @@ type KQLCustomIndicator = t.OutputOf<typeof kqlCustomIndicatorSchema>;
 export {
   createSLOParamsSchema,
   deleteSLOParamsSchema,
+  deleteSloInstancesParamsSchema,
```

Comment:
@kdelemme Shall we rename to `deleteSLOInstancesParamsSchema` to be consistent with the rest?